core/bin/block_reverter/src/main.rs

```diff
@@ -99,16 +101,16 @@ async fn main() -> anyhow::Result<()> {
             .default_priority_fee_per_gas,
     );
     let contracts = ContractsConfig::from_env().context("ContractsConfig::from_env()")?;
+    // FIXME: is it correct to parse the entire `NetworkConfig`?
```

Comment:
Subj. The code in `main` reads a specific env var; is there a particular reason to it?

https://github.com/matter-labs/zksync-era/blob/884882039c54fd301be09796c8bdd783f2c866e2/core/node/block_reverter/src/lib.rs#L349-L352

Reply:
It is a hack to manually pull a single var from what actually has to be `NetworkConfig`. The `NetworkConfig` has to be fully present on the node that we do reversion from, so it is ok to just parse it.